### PR TITLE
Redirect RHEL run network config utils sh script errors to stderr

### PR DESCRIPTION
**Pull request by yaacov**
Redirect RHEL run network config utils sh script errors to stderr

Issue:
We redirect all output from the script into the udev script file

Fix:
a. Print warning and errors to stderr
b. Redirect only stdout output into the udev file

### CODE REVIEW

File: virt-v2v/scripts/rhel/run/network_config_util.sh [hunk 1]
```diff
@@ -8,13 +8,13 @@ UDEV_RULES_FILE="${UDEV_RULES_FILE:-/etc/udev/rules.d/70-persistent-net.rules}"
 
 # Check if mapping file does not exist
 if [ ! -f "$V2V_MAP_FILE" ]; then
-    echo "File $V2V_MAP_FILE does not exist. Exiting."
+    echo "File $V2V_MAP_FILE does not exist. Exiting." >&2
```
Comment (fabiand) [inline virt-v2v/scripts/rhel/run/network_config_util.sh:11]: can we have a function like

```
err() { echo "$@" >&2 ; }
```

**Review by fabiand — CHANGES_REQUESTED** [review]

File: virt-v2v/scripts/rhel/run/network_config_util.sh (hunk 1 above)
Comment (yaacov) [reply virt-v2v/scripts/rhel/run/network_config_util.sh]: done

Comment (yaacov) [reply virt-v2v/scripts/rhel/run/network_config_util.sh]: changed to log file descriptor and redirect to stdout as @mnecas suggested

tests:
dash virt-v2v/scripts/rhel/run/network_config_util_test

File: virt-v2v/scripts/rhel/run/network_config_util.sh [hunk 2]
```diff
@@ -110,7 +116,7 @@ check_dupe_hws() {
 
     # If duplicates are found, print an error and exit
     if [ -n "$dupes" ]; then
-        echo "Warning: Duplicate hw: $dupes"
+        err "Warning: Duplicate hw: $dupes"
```
[the remark this answers is not in the document]
Comment (yaacov) [reply virt-v2v/scripts/rhel/run/network_config_util.sh]: thanks !

**Review by fabiand — COMMENTED** [review]
Can we also make a change to explicitly only redirect `echo "SUBSYSTEM==\"net\",ACTION==\"add\",ATTR{address}==\"$(remove_quotes "$S_HW")\",NAME=\"$(remove_quotes "$DEVICE")\""` to the `70-` file?

**Pull request merged**